llama_index/indices/struct_store/sql_query.py

```diff
@@ -243,17 +243,13 @@ def service_context(self) -> ServiceContext:
 
     def _parse_response_to_sql(self, response: str) -> str:
         """Parse response to SQL."""
-        # Find and remove SQLResult part
-        sql_result_start = response.find("SQLResult:")
         sql_query_start = response.find("SQLQuery:")
-        if sql_result_start != -1 and sql_query_start != -1:
-            response = response[sql_query_start + 1 : sql_result_start].lstrip(
```

Comment:
@logan-markewich actually someone talked about this in a PyCon talk this year.  Basically `lstrip` ([docs](https://docs.python.org/3/library/stdtypes.html#str.lstrip)) removes chars, not words

Reply:
huh, I always thought it meant left-strip? That explains a lot. Good catch!!

Reply:
Explains the recent errors I've seen with open-source models that are less likely to follow instructions

Reply:
Yeah it has bitten me too before.  Thanks for the PR reviews!